.jenkins

```diff
@@ -0,0 +1 @@
+.ci
```

Comment:
what does this do?

Reply:
Yeah, that how a symlink looks like on a PR, it's not a normal file though but a visual cue on where the symlink points to I think

Reply:
Ah, okay. Thanks for the explanation!